### DISCUSSION TITLE
加密与隐藏可以支持正则或者通配符吗

### DISCUSSION THREAD

**Discussion by KioProject123** [AlistGo/alist#238] [Ideas]
方便批量加密隐藏大量文件

**Comment by xhofe** [comment]
不可以 需要加密大量文件将文件放入一个文件夹加密这个文件夹打开[check parent password](https://alist-doc.nn.ci/docs/setting/private#%E6%A3%80%E6%9F%A5%E7%88%B6%E6%96%87%E4%BB%B6%E5%A4%B9%E5%AF%86%E7%A0%81)就好了
> 启用后，如果当前文件夹未设置密码，会依次检查父文件夹是否有密码，直至检查到第一个密码或到根目录停止。
